### PR TITLE
Add Websocket Configuration Documentation

### PR DESCRIPTION
**Pull request by ColinSullivan1**
Signed-off-by: Colin Sullivan <colin@synadia.com>

### CODE REVIEW

File: nats-server/configuration/websockets.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    # If same_origin is true, then the Origin header of the
+    # client request must match the request's Host.
+    # same_origin: true
+
+    # This list specifies the only accepted values for
+    # the client's request Origin header. The scheme,
+    # host and port must match. By convention, the
+    # absence of port for an http:// scheme will be 80,
```
Comment (gcolliso) [inline nats-server/configuration/websockets.md:39]: What does "...the absence of port..." mean in this context?

**Comment by aricart** [comment]
Sorry I had not pushed my changes in - https://github.com/nats-io/nats.docs/pull/217 - you can choose which ever PR you think answers it best. Perhaps just copying the configs that are there.

File: nats-server/configuration/websockets.md (hunk 1 above)
Comment (kozlovic) [reply nats-server/configuration/websockets.md:39]: I am going to update this PR with some missing params and fix that.

**Comment by kozlovic** [comment]
I have made some updates.

**Comment by ColinSullivan1** [comment]
Thanks @aricart and @kozlovic!

**Comment by kozlovic** [comment]
@ColinSullivan1 @gcolliso I just added a note about the frame types and the expectation of the content of a frame for writers of client libraries.

**Comment by kozlovic** [comment]
Resolves #205

**Comment by kozlovic** [comment]
@ColinSullivan1 @gcolliso I have changes a bit the layout to match some other features (such as leafnode). So I made a `websocket/` directory that contains a README.md and a websocket_conf.md files. I have updated all links hopefully.

**Pull request merged**